src/constraints.jl

```diff
@@ -727,3 +727,313 @@ function list_of_constraint_types(model::Model)
     return Tuple{DataType, DataType}[(jump_function_type(model, f), s)
                                      for (f,s) in list]
 end
+
+
+"""
+    perturbation_range_of_feasibility(constraint::ConstraintRef)
+
+Gives the range by which the rhs coefficient can change and the current lp-basis
+remains feasible, i.e., where the shadow prices apply.
+
+## Notes
+- The range denote valid changes, e.g., for a*x <= b + Δ, the lp-basis remains feasible for all Δ in [l, u].
+
+"""
+function perturbation_range_of_feasibility(constraint::ConstraintRef{Model, <:_MOICON})
+    error("The range of feasibility is not defined or not implemented for this type " *
+          "of constraint.")
+end
+
+"""
+Builds the standard form constraint matrix, varaible bounds (rhs := 0, by choice of slack bounds), and a vector of constraint reference one for each row.
+i.e., the LP-problem feasible set on the form
+s.t.  Ax == 0
+L <= x <= U
+"""
+function _std_matrix(model::Model)
```

Comment:
That would indeed be neat. However, I can't get it to work, I might be missing some thing (not so used with automatic bridges...)

Main issue: Every constraint should get an associated slack variable.
My idea was to copy the model to a model having only interval constraints, (Since you can create an MOI.Interval, from other the other sets.) However, I always get an unsupported constraint error. Moreover, I can't find a bridge that creates intervals (only the ones that splits intervals). 

If this was accomplished then the slack bridge would accomplish the rest (I think).

Reply:
No there is not any bridge that transforms `GreaterThan` or `LessThan` to `Interval` yet. You can try with
```julia
MOIU.@model(MyModel,
    (),
    (MOI.Interval, MOI.GreaterThan, MOI.LessThan, MOI.EqualTo),
    (),
    (),
    (MOI.SingleVariable,),
    (),
    (MOI.Zeros,),
    (MOI.VectorAffineFunction)
)
```